pkg/datapath/connector/veth.go

```diff
@@ -123,6 +123,28 @@ func SetupVethWithNames(lxcIfName, tmpIfName string, mtu int, ep *models.Endpoin
 		return nil, nil, fmt.Errorf("unable to bring up veth pair: %s", err)
 	}
 
+	if groMaxSize > 0 {
+		if err = netlink.LinkSetGROMaxSize(hostVeth, groMaxSize); err != nil {
+			return nil, nil, fmt.Errorf("unable to set GRO max size to %q: %s",
```

Comment:
I would use error wrapping for these, same goes for the 3 returns with fmt.Errorf below.
```suggestion
			return nil, nil, fmt.Errorf("unable to set GRO max size to %q: %w",
```

Reply:
ack, changed